### DISCUSSION TITLE
Disable autolinks for fenced code blocks

### DISCUSSION THREAD

**Discussion by max-arnold** [jgm/pandoc#10219] [Q&A] [answered]
Is it possible to disable autolinks in fenced `djangotemplate` code blocks?  I.e. instead of this:

```
echo -e '```djangotemplate\nhttp://example.com\n```' | pandoc -f markdown

<div class="sourceCode" id="cb1"><pre
class="sourceCode djangotemplate"><code class="sourceCode djangotemplate"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>http://example.com</span></code></pre></div>
```

get something like this:

```
echo -e '```djangotemplate\nhttp://example.com\n```' | pandoc -f markdown

<div class="sourceCode" id="cb1"><pre
class="sourceCode djangotemplate"><code class="sourceCode djangotemplate"><span id="cb1-1">http://example.com</span></code></pre></div>
```

I really do not want any links within code snippets to be clickable...

**Comment by jgm** [comment, marked as answer]
No, there's no way to avoid this.  You could strip it out with JS.
(Anyway, there won't be any visible clickable links unless you specify line numbers.)

**Reply by max-arnold** [reply]
Even custom filters can't do this?

Does it make sense to submit a feature reqest, or it is unlikely to be implemented?

**Reply by jgm** [reply]
You could submit something to jgm/skylighting which does our highlighting.
I don't know whether it would be implemented.  I seem to recall there was a reason for putting these there, even without the line numbers.
